Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReport.c

```diff
@@ -1543,77 +1549,99 @@ void bsg_kscrashreport_writeStandardReport(
     bsg_ksjsonbeginEncode(bsg_getJsonContext(writer), true,
                           bsg_kscrw_i_addJSONData, &fd);
 
+    bool recordKSCrashFields = crashContext->crash.userException.eventOverrides == NULL;
```

Comment:
This feels like a good point to add some comments about why different fields are included/excluded.

Reply:
Added inline comment.